packages/fxa-react/components/AppErrorDialog/index.tsx

```diff
@@ -2,26 +2,56 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { Localized } from '@fluent/react';
+import { error } from 'console';
+
+export type ErrorType = 'general' | 'query-parameter-violation';
+
+const AppErrorDialog = ({
+  errorType,
+  errors,
+}: {
+  errorType?: ErrorType;
+  errors?: ReactNode;
+}) => {
+  if (errorType == null) {
+    errorType = 'general';
+  }
 
-const AppErrorDialog = () => {
   return (
-    <div className="bg-grey-20 flex items-center flex-col justify-center h-screen">
```

Comment:
`h-screen` is needed to centre the error message vertically

Reply:
Okay, I'll add this back in. Fwiw, I was getting some of the the dialog below the folding line when window height was reduced. I'm guessing there's also a min height somewhere. Seems pretty minor either way, and I'd rather it be consistent.